### ISSUE TITLE
Could i am allowed to connect to Radius with my other application. error ignoring request to auth address * port 1812 bound to server default from unknown client 

### ISSUE THREAD

**Issue by KamranShahid** [frauhottelmann/daloradius-docker#16]
I have a .net core application from which i am successfully able to connect and authenticate with freeradius docker container .
But freeradius doesn't have such user interface from which i can add or manage something.
So i am trying daloradius. i am able to setup and run the Userinterface and added user.
Same application from which i am able to authenticate from freeradius is not able to connect with ip of daloradius.
Is this possible? I am able to connect to database and see things from my system but radius command not working with my .net core app.
I am even not able to telnet to port 1812/1813 with the assigned ip

**Comment by KamranShahid** [comment]
I have tried with attached docker-compose file as well
[docker-compose.zip](https://github.com/frauhottelmann/daloradius-docker/files/7642286/docker-compose.zip)

**Comment by frauhottelmann** [comment]
Mh, not sure why you would be able to reach the UI, but not freeradius itself. It's only UDP. Can you check whether free radius is running inside the container? Maybe remove UDP from.the ports? You can test with radtest.

**Comment by KamranShahid** [comment]
> Mh, not sure why you would be able to reach the UI, but not freeradius itself. It's only UDP. Can you check whether free radius is running inside the container? Maybe remove UDP from.the ports? You can test with radtest.

i have shared my docker compose above. any hint from it?

**Comment by KamranShahid** [comment]
What i further see is my portQryUI is able to connect to udp port 1812 (both freeradius and daloradius)

Problem is further in my application i am getting socket exception when trying to connect

A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
SocketTimeout happeneing

My Auth request some attributes are

 IP = "127.0.0.1",
secret = "testing123",
username = "testing",
password = "password",

**Comment by KamranShahid** [comment]
From interface when i have tried authentication then it's work. is there anyway i can increase timeout to connect?

**Comment by KamranShahid** [comment]
what I saw is it looks like app is ignoring call from random ip in as per docker logs. Any idea how to allow call.
In logs following is coming  "Ignoring request to auth address * port 1812 bound to server default from unknown client 11.0.0.1 port 59849 proto udp"
About allowing Client IP I saw freeradius docker file we have a variable

ENV RAD_CLIENTS=11.0.0.0/24  in which we can allow ip's or range of ip's

Any such thing we can do here?
